docs/documentation/pack-structure.md

```diff
@@ -20,7 +20,7 @@ show_toc: false
     'BP/recipes/example.recipe.json',
     'BP/scripts/client/exampleClient.js',
     'BP/scripts/server/exampleServer.js',
-    'BP/scripts/gametests/exampleTest.js',
+    'BP/scripts/exampleTest.js',
```

Comment:
This should be change to `BP/scripts/exampleScript.js` since it isn't just for tests

Reply:
Updated the PR to fix this issue.